zerver/lib/actions.py

```diff
@@ -944,11 +944,17 @@ def send_welcome_bot_response(message: MutableMapping[str, Any]) -> None:
     welcome_bot = get_system_bot(settings.WELCOME_BOT)
     human_recipient_id = message['message'].sender.recipient_id
     if Message.objects.filter(sender=welcome_bot, recipient_id=human_recipient_id).count() < 2:
+        content = (
+            _("Congratulations on your first reply!") +
+            " "
+            ":tada:"
+            "\n"
+            "\n" +
+            _("Feel free to continue using this space to practice your new messaging \
+            skills. Or, try clicking on some of the stream names to your left!")
```

Comment:
I'm not a fan of the line-wrapping strategy in this last line; we should just use ` "\n"` instead and use Python's automatic concatenation of strings.  That's a lot more explicit about where spaces will appear in the result.

Also, can you add an extra commit moving this function to `onboarding.py`?  It clearly belongs there.